collector/meminfo_numa_linux.go

```diff
@@ -93,6 +95,21 @@ func getMemInfoNuma() (map[meminfoKey]float64, error) {
 		for k, v := range numaInfo {
 			info[k] = v
 		}
+
+		file, err := os.Open(path.Join(node, "numastat"))
```

Comment:
Since you're re-using both variables here, make this just `=`. Or use a new variable for a new file.